src/components/ValueSet/ValueSetForm.tsx

```diff
@@ -340,8 +340,15 @@ function RuleFields({
 }
 
 const valuesetFormSchema = z.object({
-  name: z.string().min(1, t("field_required")),
-  slug: z.string().min(1, t("field_required")),
+  name: z.string().trim().min(1, t("field_required")),
+  slug: z
+    .string()
+    .trim()
+    .min(5, t("field_required"))
+    .max(25, t("slug_lengthy_message"))
```

Comment:
let's use variables to keep the message generic so that it can be re-used elsewhere instead of creating too many translations keys specific for each one

```suggestion
    .max(25, t("max_character_validation", { length: 25 }))
```

Reply:
Complete the merge checklist as well @hrit2773

Reply:
Complete the merge checklist as well @hrit2773

Reply:
> Complete the merge checklist as well @hrit2773

Yup